src/configure/configurers/localGithubWorkflowConfigurer.ts

```diff
@@ -274,10 +274,10 @@ export class LocalGitHubWorkflowConfigurer implements Configurer {
                 let authorName = await LocalGitRepoHelper.GetHelperInstance(inputs.sourceRepository.localPath).getUsername();
                 let deployerName = 'GITHUBACTION';
                 let updateDeploymentLogPromise = (azureResourceClient as AppServiceClient).publishDeploymentToAppService(inputs.targetResource.resource.id, deploymentMessage, authorName, deployerName);
-
-                Q.all([updateMetadataPromise, updateDeploymentLogPromise])
+                Q.all([sourceControlPromise, updateDeploymentLogPromise])
                     .then(() => {
                         telemetryHelper.setTelemetry(TelemetryKeys.UpdatedWebAppMetadata, 'true');
+                        telemetryHelper.setTelemetry(TelemetryKeys.UpdateWepAppSourceControl, 'true');
```

Comment:
I think we should just have one telemetry to log. Does not make sense to log two things for same point in code execution.
May be it was okay the way it was.